modules/geolocation/geolocation.py

```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 
+
```

Comment:
Avoid adding empty lines, you should at most have one empty line between two populated lines

Reply:
I believe two lines between functions is the standard.